infrastructure/code/sast_local_engine.go

```diff
@@ -0,0 +1,50 @@
+/*
+ * © 2023 Snyk Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package code
+
+import (
+	"github.com/rs/zerolog/log"
+	sglsp "github.com/sourcegraph/go-lsp"
+
+	"github.com/snyk/snyk-ls/application/config"
+	"github.com/snyk/snyk-ls/infrastructure/snyk_api"
+)
+
+const localEngineMisConfiguredMsg = "Snyk Code Local Engine (SCLE) is enabled but the SCLE URL is not configured. Read our docs on how you can configure the SCLE URL https://docs.snyk.io/products/snyk-code/deployment-options/snyk-code-local-engine/cli-and-ide"
```

Comment:
Is the url clickable/navigatable from IDE side, or a user might experience pain in copy-pasting it to a browser? If latter, perhaps we can use SendShowRequestMessage via notifier, to create a button to open a browser with the url.

Reply:
@metju90 did you see this in IDE already? is it something we can test before merging?

Reply:
The button would just need to refer to the snyk.OpenBrowserCommand

Reply:
@michelkaporin @bastiandoetsch tnx for the improvement - indeed the URL was not clickable.  I have added this functionality to the code now! looks ok on IDE when I tested it, you can review it now